### PR TITLE
Example StorageClass for Ceph External cluster

### PR DESCRIPTION
**Pull request by alexcpn**
**Description of your changes:**
Documentation changes, improving usability for testing external ceph cluster feature

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [x ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [x] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
[skip ci]

### CODE REVIEW

File: Documentation/ceph-cluster-crd.md [hunk 1]
```diff
@@ -971,17 +979,70 @@ spec:
...
+If this is successfull you will see the CepCluster status as connected.
 
-You can now create it like this:
+```console
+kubectl get CephCluster -n rook-ceph-external
+NAME                 DATADIRHOSTPATH   MONCOUNT   AGE    STATE       HEALTH
+rook-ceph-external   /var/lib/rook                162m   Connected   HEALTH_OK
+```
+
+You can not create a StorageClass with this cluster. Before that you will need to create a Pool in your external Ceph Cluster.
```
Comment (travisn) [inline Documentation/ceph-cluster-crd.md:992]:
```suggestion
Before you create a StorageClass with this cluster you will need to create a Pool in your external Ceph Cluster. 
```

File: Documentation/ceph-cluster-crd.md [hunk 2]
```diff
@@ -948,6 +948,14 @@ Finally, you can simply execute the script like this from a machine that has acc
 ```console
 bash cluster/examples/kubernetes/ceph/import-external-cluster.sh
 ```
+Choose the namespace carefully, if you have an existing cluster managed by Rook, you have likely already injected `common.yaml`.
```
Comment (travisn) [inline Documentation/ceph-cluster-crd.md:951]: We only need to create common-external.yaml if there is a different namespace than `rook-ceph` or if you want to run both a local cluster and an external cluster, right? But I'm still not clear from this section. The main scenario should be to only connect to the external cluster. It should be uncommon to have a local cluster at the same time except for testing purposes. Let's focus this doc on the scenario of only having an external cluster. How does that change this paragraph?

**Comment by travisn** [comment]
@alexcpn Also see the three build changes needed: commitlint and DCO are for the commit message. For gosec you can rebase to pick up the fix.

File: Documentation/ceph-cluster-crd.md (hunk 2 above)
Comment (alexcpn) [reply Documentation/ceph-cluster-crd.md:951]: I am not sure if it is uncommon . For instance, we have a storage cluster that we have external ceph on , along with internal ceph.  But I agree this maybe not so common. If you feel so I will remove it and add only the Storage yaml example.

**Comment by alexcpn** [comment]
@travisn reopened here with review comments updated https://github.com/rook/rook/pull/5946

**Comment by travisn** [comment]
> @travisn reopened here with review comments updated #5946

@alexcpn ok thanks, in the future it would be preferred if you could just make the updates, squash, and force push to the same PR

**Pull request closed without merging**